### DISCUSSION TITLE
Site not rendering correctly at deployment

### DISCUSSION THREAD

**Discussion by ecbettinger** [CollectionBuilder/collectionbuilder.github.io#93] [Q&A]
We have a site (https://waleskadelsol.github.io/venesporanartists_source/) which is not deploying partially correctly (Object pages appear correctly), but has major sections (Data Table, Carousel, Map) not rendering at all. We tried hosting on Render as well, with the same results, which makes us think that the problem is not host-specific. 

Any ideas for where to look for help? 
Please note that many of the objects do NOT have images associated with them, though two do, including those included in the carousel on the front page. 

Thanks so much in advance for any tips you can offer!

**Comment by evanwill** [comment]
@ecbettinger the issue with your pages is that you modified the "_layouts/default.html" removing `{% include foot.html %}`. The foot include doesn't add anything directly visible, but it adds all the javascript assets and code for all pages (this is a method to optimize adding js to the page load)--without the foot.html none of your pages will function or load correctly (as you found)! 
If you restore the file "_layouts/default.html" in your project to match the default [CB-CSV version](https://github.com/CollectionBuilder/collectionbuilder-csv/blob/main/_layouts/default.html) then everything will start working again!

**Reply by ecbettinger** [reply]
Thank you! We will take a look at this!!
